File: 백준/Bronze/10430. 나머지/나머지.js
const fs = require('fs');
const input = fs.readFileSync('/dev/stdin').toString().split(' ');
const [A, B, C] = [Number(input[0]), Number(input[1]), Number(input[2])];
console.log ((A + B) % C);
console.log (((A % C) + (B % C)) % C);
console.log ((A * B) % C);
console.log (((A % C) * (B % C)) % C);

File: 백준/Bronze/10430. 나머지/나머지.py
import sys

a, b, c = [int(i) for i in sys.stdin.readline().split()]

results = [(a + b) % c, ((a % c) + (b % c)) % c, (a * b) % c, ((a % c) * (b % c)) % c]
for i in results:
    print(i)